packages/file-upload/package.json

```diff
@@ -10,17 +10,19 @@
 		"react-dropzone": "^12.0.5"
 	},
 	"devDependencies": {
-		"@ag.ds-next/box": "*",
-		"@ag.ds-next/button": "*",
-		"@ag.ds-next/core": "*",
-		"@ag.ds-next/field": "*",
-		"@ag.ds-next/icon": "*",
-		"@ag.ds-next/loading": "*",
-		"@ag.ds-next/text": "*",
+		"@ag.ds-next/a11y": "1.2.1",
```

Comment:
Can we revert these back to `*` please just so it's consistent with other packages.